Resources/Prototypes/_NF/Procedural/chromite_vgroid.yml

```diff
@@ -272,12 +272,12 @@
     groups:
     - id: SpawnMobBloodCultistAscended # Boss
       amount: 1
-  - !type:MobsDunGen
-    minCount: 1
-    maxCount: 1
-    groups:
-    - id: SpawnMobCatBloodCultLowChance
-      amount: 1
+#  - !type:MobsDunGen # Cat dies to space, big sad
+#    minCount: 1
+#    maxCount: 1
+#    groups:
+#    - id: SpawnMobCatBloodCultLowChance
+#      amount: 1
```

Comment:
Could either remove the respirator (it's a magic spooky cat, mistake could probably work the same way) or give it internals via a loadout.